common/consts/consts.go

```diff
@@ -47,6 +47,12 @@ const (
 	InstrumentationConfig   = "InstrumentationConfig"
 	InstrumentationInstance = "InstrumentationInstance"
 	Destination             = "Destination"
+
+	// ConfigMap used to store custom/updated Go instrumentation offsets
+	GoOffsetsPublicURL = "https://storage.googleapis.com/odigos-cloud/offset_results.json"
+	GoOffsetsConfigMap = "odigos-go-offsets"
+	GoOffsetsFileName  = "offset_results.json"
+	GoOffsetsEnvVar    = "OTEL_GO_OFFSETS_FILE"
```

Comment:
as these are k8s specific, they belong in the api module `api/k8sconsts`, the `common` module is shared with vmagent.